### PR TITLE
Edge polling

### PR DESCRIPTION
**Pull request by likewei92**
Makes controller polling work on edge/xbox

### CODE REVIEW

File: src/input.service.ts [hunk 1]
```diff
@@ -455,9 +457,10 @@ export class InputService {
    * a connected gamepad somewhere.
    */
   private pollGamepad(now: number) {
-    navigator.getGamepads(); // refreshes all checked-out gamepads
+    const rawpads = navigator.getGamepads(); // refreshes all checked-out gamepads
 
     for (let i = 0; i < this.gamepads.length; i += 1) {
+      this.gamepads[i].pad = rawpads[i];
```
Comment (connor4312) [inline src/input.service.ts]: You have to make sure that the gamepad is the one assigned to that specific instance. If someone disconnects a controller the indicies may change.

File: src/input.service.ts [hunk 2]
```diff
@@ -42,6 +42,8 @@ interface IGamepadWrapper {
    * Returns whether the gamepad is still connected;
    */
   isConnected(): boolean;
+
+  pad: Gamepad;
```
Comment (connor4312) [inline src/input.service.ts]: Docblock

Comment (likewei92) [inline src/input.service.ts]: what's docblock?

Comment (MattJustMatt) [inline src/input.service.ts]: He's asking for a comment about what "pad"s role is in this context

Comment (likewei92) [inline src/input.service.ts]: ok added

**Pull request merged**